src/common/config.ts

```diff
@@ -1,9 +1,18 @@
-import fs = require('fs');
-import path = require('path');
+import fs = require("fs");
+import path = require("path");
+import { configureLogger } from "./configureLogger";
 
-const configFilePath = path.resolve(__dirname,'../../config/config.json')
-const configBuffer: string = fs.readFileSync(configFilePath,{ encoding: 'utf8', flag: 'r'});
+const configFilePath = path.resolve(__dirname, "../../config/config.json");
+const configBuffer: string = fs.readFileSync(configFilePath, {
+  encoding: "utf8",
+  flag: "r",
+});
 
 export const config: Record<string, any> = JSON.parse(configBuffer);
-console.log("Configuration");
-console.log(config)
+
+configureLogger(
```

Comment:
Is this file the correct place for the logger configuration?
I would put it in the app.ts or index.ts